### PR TITLE
Story Tool Design Updates

### PR DESCRIPTION
**Pull request by mzagaja**
* Update banner to comport with mockup
* Modify button placement and styles to comport with mock up suggestions

### CODE REVIEW

File: app/assets/stylesheets/refinery/stories.scss [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
   a.button { text-decoration: none; }
 
+  .audio-response, .video-response {
+    display: inline-block;
+    vertical-align: top;
+    margin-right: 2rem;
+  }
+
+  .banner {
+    background: $color_bg-dark image-url('story-tool-banner.png');
```
[the remark this answers is not in the document]
Comment (mzagaja) [reply app/assets/stylesheets/refinery/stories.scss]: What approach would you suggest here?

**Comment by mzagaja** [comment]
Added in the 'or' block between the two buttons. It was suggested in the design but I gave up since thought I was just trying to ship it before like 4 and didn't want to futz with it since it was being finicky.

File: app/assets/stylesheets/refinery/stories.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericyoungberg) [reply app/assets/stylesheets/refinery/stories.scss]: Doing it with CSS. The image way means you have to maintain two images (1x and 2x), you have to export new images when the site brand changes colors throughout the phases, and the user loads an additional ~414,000 bytes where CSS could do it in < 400 bytes.

At the least, remove the background from the PNG so the only things in the image are the triangles; the background color difference is the biggest visual issue.

**Comment by mzagaja** [comment]
Ok pushed new PNGs and made backgrounds transparent.

**Comment by ericyoungberg** [comment]
Noice 

screen shot 2019-02-01 at 3 11 39 pm

**Pull request merged**